client/resources/aria-at-harness.mjs

```diff
@@ -430,55 +448,81 @@ function renderVirtualInstructionDocument(doc) {
         className(['problem-select']),
         id(`cmd-${commandIndex}-problem-checkboxes`),
         legend(rich(unexpected.failChoice.options.header)),
-        ...unexpected.failChoice.options.options.map(failOption =>
-          fragment(
-            input(
-              type('checkbox'),
-              value(failOption.description),
-              id(`${failOption.description}-${commandIndex}`),
-              className([`undesirable-${commandIndex}`]),
-              tabIndex(failOption.tabbable ? '0' : '-1'),
-              disabled(!failOption.enabled),
-              checked(failOption.checked),
-              focus(failOption.focus),
-              onchange(ev =>
-                failOption.change(/** @type {HTMLInputElement} */ (ev.currentTarget).checked)
+        ...unexpected.failChoice.options.options.map(failOption => {
+          const failOptionId = failOption.description
+            .toLowerCase()
+            .replace(/[.,]/g, '')
+            .replace(/\s+/g, '-');
+
+          return div(
+            className([`problem-option-container ${failOption.checked ? 'enabled' : ''}`]),
+            // Undesirable behavior checkbox
+            div(
```

Comment:
I appreciate your sensitivity to the size of this `return` statement, but rather than inserting comments to help explain what's going on, my preference would be to use a more self-documenting style: a series of `const` statements defining the values, and a streamlined `return` statement which references those values. I'm just a visitor here, though, so I'll leave this judgement call in your hands.

Reply:
I was mostly following the existing style of this particular file, which seems to prefer inlining element declarations (seemingly to make the rendered HTML structure easier to understand?) and only using `function`s or `const`s where inline elements could be used more than once. But with that, I do still think this suggestion falls in line with how this particular file's functions were done and I prefer it.

Change done at https://github.com/w3c/aria-at/pull/1016/commits/8e5b92a6492863af0d3a28d0194a4e73bcd9bfd4, and will be moved over to here.